GrocifyApp.BLL/Implementations/InventoryService.cs

```diff
@@ -0,0 +1,83 @@
+﻿using GrocifyApp.BLL.Data.Consts.ENConsts;
+using DALConsts = GrocifyApp.DAL.Data.Consts.ENConsts;
```

Comment:
please review Delete here and ShoppingList, when deleting an entity that is the default, it should make other be the default

Reply:
Can you explain?